src/pages/onboarding/components/steps/optionGroup/MultiOptionGroup.tsx

```
@@ -75,21 +78,20 @@ const MultiOptionGroup = <T = string,>({
 
       <div className={styles.buttonBox}>
         {options.map((option) => {
+          const optionValue = useId ? option.id : option.code;
           const isRequired = isRequiredFurniture
-            ? isRequiredFurniture(option.code)
+            ? isRequiredFurniture(optionValue!)
             : false;
-          const isSelected = selected.includes(option.code);
+          const isSelected = selected.includes(optionValue!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)